plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -475,4 +477,30 @@ public DeltaLakeConfig setRegisterTableProcedureEnabled(boolean registerTablePro
         this.registerTableProcedureEnabled = registerTableProcedureEnabled;
         return this;
     }
+
+    @Config("delta.reader-version")
+    @ConfigDescription("The reader version used by tables created by Trino")
+    public DeltaLakeConfig setReaderVersion(int readerVersion)
+    {
+        this.readerVersion = readerVersion;
+        return this;
+    }
+
+    public int getReaderVersion()
+    {
+        return readerVersion;
+    }
+
+    @Config("delta.writer-version")
+    @ConfigDescription("The writer version used by tables created by Trino")
+    public DeltaLakeConfig setWriterVersion(int writerVersion)
+    {
+        this.writerVersion = writerVersion;
+        return this;
+    }
+
+    public int getWriterVersion()
+    {
+        return writerVersion;
+    }
```

Comment:
Replace getter and setter order for consistency with other properties.

Reply:
setter-getter is inconsistent in this file (setter-getter then getter-setter). What is the ideal way?

Reply:
At least, my IntelliJ generates getter-setter.